spec/factories/articles.rb

```diff
@@ -25,6 +25,7 @@
     description { Faker::Hipster.paragraph(sentence_count: 1)[0..100] }
     main_image    { with_main_image ? Faker::Avatar.image : nil }
     experience_level_rating { rand(4..6) }
+    # cached_tag_list { tags if with_tags }
```

Comment:
I probably shouldn't have committed this comment if it's not needed. will remove after the CI build